sources/packages/api/src/route-controllers/institution/models/institution-datatable.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Enumeration for DB Sort order
+ */
+export enum FieldSortOrder {
+  DESC = "DESC",
+  ASC = "ASC",
+}
+/**
+ * Enum for Institution User Table
+ */
+export enum UserFields {
```

Comment:
We can give a more closer name like UserSortFields

Reply:
These are not sort fields, these are user tables field names

Reply:
IMO, these are UI field names and is this really required to have them all in a structure? As per the part of the API, it should work with any dummy values as string in the queryparam for sortField, and handled at the service layer in the else block. Please let me know your thoughts.

Reply:
updated as discussed